app/models/iiif_content_search_response.rb

```diff
@@ -30,7 +30,7 @@ def resources
     search.highlights.each do |id, hits|
       # Hit here refers to the Solr hit
       hits.each do |hit|
-        hit.to_enum(:scan, %r{<em>.*?</em>}).each do |*_match|
+        hit.gsub('</em> <em>', ' ').to_enum(:scan, %r{<em>.*?</em>}).each do |*_match|
```

Comment:
What will this in the case that a non-highlighted word shows up between highlighted words?
e.g. 
```html
<em>crimes</em> <em>are</em> bad <em>mkay?</em>
```

Reply:
I think it will be fine here and give something like the following:

```html
<em>crimes are</em> bad <em>mkay?</em>
```

Which then gets enumerated in the each (to two matches) by the scan of `%r{<em>.*?</em>}` right?

Reply:
Yeah.  That makes sense.  Thanks, was just trying to wrap my head around that.